config/routes.rb

```diff
@@ -2,5 +2,5 @@
 
 Rails.application.routes.draw do
   resources :movies, only: %i[index show]
-  resources :cinema_halls, only: %i[index show]
+  resources :cinema_halls
```

Comment:
Remember that when you not specifying `only` or `except` params, you will have a lot of useless endpoints created here - especially non-api related, like `new`, `delete`, `edit` etc. Consider adding here:

`only: %i[index show update create destroy]`